standard/openapi/examples/collection.json

```diff
@@ -50,31 +50,28 @@
                     "type": "text/html"
                 },
                 {
-                    "href": "https://www.metoffice.gov.uk/services/data/datapoint/terms-and-conditions---datapoint#termsofservice",
+                    "href": "http://www.example.org/data/datapoint/terms-and-conditions---datapoint#termsofservice",
                     "hreflang": "en",
                     "rel": "restrictions",
                     "type": "text/html"
                 },
                 {
-                    "href": "http://www.example.org/edr/collections/hrly_obs/position",
+                    "href": "http://www.example.org/edr/collections/hrly_obs?f=json",
                     "hreflang": "en",
-                    "rel": "data"
```

Comment:
Same here, no relation type for this link?

Reply:
@tomkralidis, they should be collection responses I have updated the pull request